ui/components-react/src/test/filter-builder/FilterBuilderState.test.ts

```diff
@@ -342,12 +368,70 @@ describe("usePropertyFilterBuilderState", () => {
     });
 
     it("removes group when last rule is removed", () => {
-      const {result, getNestingRule, getNestedRulePath} = getStateWithNestedRule();
-      const {actions} = result.current;
+      const { result, getNestingRule, getNestedRulePath } = getStateWithNestedRule();
+      const { actions } = result.current;
 
       actions.removeItem(getNestedRulePath());
 
       expect(getNestingRule()).to.be.undefined;
     });
   });
 });
+
+describe("conver to filter builder state", () => {
```

Comment:
I don't think we need tests for `convertFilterToState` function as it only adds `id` and `parentId` properties and should be internal to `PropertyFilterBuilder` component. I think adding tests that check if `PropertyFilterBuilder` component is rendered correctly with initialFilter would cover this function.